src/api.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+import { Todo } from './types/Todo';
+
+const BASE_URL = 'https://mate.academy/students-api';
+
+export const getTodosFromServer = (userId: number, param = '') => {
+  return fetch(`${BASE_URL}/todos?userId=${userId}&${param}`)
+    .then((data) => data.json())
+    .catch((error) => new Error(error.message));
+};
+
+export const getCompetedTodosFromServer = (userId: number) => {
+  fetch(`${BASE_URL}/todos?userId=${userId}&completed=true`)
+    .then((data) => data.json())
+    .catch((error) => new Error(error.message));
+};
+
+export const setTodoOnServer = (srt: string, userId: number) => {
+  return fetch(`${BASE_URL}/todos`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json; charset=utf-8',
+    },
+    body: JSON.stringify({
+      userId,
+      completed: false,
+      title: srt,
+    }),
+  }).catch((error) => console.log('add', error.message));
+};
+
+export const deleteTodoOnServer = (id: number) => {
+  return fetch(`${BASE_URL}/todos/${id}`, {
+    method: 'DELETE',
+  }).catch((error) => new Error(error.message));
+};
+
+export const setTodoCompleteStatus = (id: number, object: Partial<Todo>) => {
```

Comment:
```suggestion
export const updateTodo = (id: number, object: Partial<Todo>) => {
```
Consider renaming to `updateTodo`, this function can be used to update other fields, not only `completed`